src/utils/.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Um, I don't think we want this eslintrc here. the src/utils folder should not be importing dev dependencies.